packetbeat/protos/tls/tls.go

```diff
@@ -248,8 +248,7 @@ func (plugin *tlsPlugin) sendEvent(conn *tlsConnectionData) {
 	if !conn.eventSent {
 		conn.eventSent = true
 		if conn.hasInfo() {
-			event := plugin.createEvent(conn)
-			plugin.results(event)
+			plugin.results(plugin.createEvent(conn))
```

Comment:
The linter claimed `event` was declared but unused. Not sure why. But this change fixed made it happy.